index.html

```diff
@@ -35,7 +38,10 @@
       even run your own local version of it.
     </div>
   </div>
-  <h2>Getting started</h2>
+  <h2>
+    {% icon fa-heart-o %}
```

Comment:
I am not a big fan of the `fa-heart-o` for "Getting started". How do you feel about `fa-truck` or `fa-flag-o` or maybe an arrow? I think those better represent something that is starting/moving.